components/app.js

```diff
@@ -1,14 +1,23 @@
 class App {
-  constructor(gradeTable) {
+  constructor(gradeTable, pageHeader) {
     this.gradeTable = gradeTable;
+    this.pageHeader = pageHeader;
     this.handleGetGradesError = this.handleGetGradesError.bind(this);
     this.handleGetGradesSuccess = this.handleGetGradesSuccess.bind(this);
   }
   handleGetGradesError(error) {
     console.error(error);
   }
   handleGetGradesSuccess(grades) {
+    var gradeAverage = 0;
     this.gradeTable.updateGrades(grades);
+    for (let i = 0; i < grades.length; i++) {
+      gradeAverage = (gradeAverage+grades[i].grade)
+      if (i === grades.length-1) {
```

Comment:
Rather than doing this with a conditional inside of your loop, you can simply do these actions after your loop has stopped, since that is basically what your condition is checking for 👍

Reply:
![image](https://user-images.githubusercontent.com/61394367/81337202-7fe18380-905f-11ea-871d-fcdafa67aa6a.png)